clients/telemetry/src/main/java/org/eclipse/hono/client/telemetry/EventSender.java

```diff
@@ -12,7 +12,7 @@
  */
 
 
-package org.eclipse.hono.adapter.client.telemetry;
+package org.eclipse.hono.client.telemetry;
```

Comment:
How about updating copyright notice with current year?

Reply:
well, I haven't changed the classes but merely renamed the package. But I am glad to adapt the year as well ...